addons/mail/static/src/js/basic_view.js

```diff
@@ -1,18 +1,27 @@
 odoo.define('mail.BasicView', function (require) {
 "use strict";
 
-var BasicView = require('web.BasicView');
+const BasicView = require('web.BasicView');
 
-var mailWidgets = ['mail_followers', 'mail_thread', 'mail_activity', 'kanban_activity'];
+const supportedMailFields = {
```

Comment:
name suggestion
```suggestion
const supportedChatterFields = {
```
Note that at the end (and maybe as part of this PR since we have diff on all of them anyway) we probably want to change the syntax so that it does not depend on fields (since in reality it doesn't) but with a way to simply "pass params" to `oe_chatter`.

Need to change in accordance with AL, but he was positive about the idea last time I asked.

Reply:
Yep I agree but as you said we should maybe see the "public API" with AL